nginx/etc/nginx/conf.d/default.conf

```diff
@@ -0,0 +1,61 @@
+
+server {
+    listen 80;
+    server_name staging.rasterfoundry.com rasterfoundry.com;
+    return 301 https://$host$request_uri;
+}
+
+map $http_x_forwarded_proto $policy {
+    default "";
+    https   "default-src https: data: blob: 'unsafe-inline' 'unsafe-eval'";
+}
+
+upstream app-server-upstream {
+    server app-server:9000;
+}
+
+server {
+    listen 443 default_server;
+    server_name staging.rasterfoundry.com rasterfoundry.com localhost;
+
+    # A set of recommended security headers:
+    #
+    #   https://scotthelme.co.uk/hardening-your-http-response-headers/
+    #
+    add_header Strict-Transport-Security "max-age=15552000; preload" always;
+    add_header Content-Security-Policy $policy always;
+    add_header X-Content-Type-Options "nosniff" always;
+    add_header X-Frame-Options "SAMEORIGIN" always;
+    add_header X-XSS-Protection "1; mode=block" always;
+
+    # Akka-Http API
+    location /api {
+        proxy_set_header Host $http_host;
+        proxy_set_header X-Forwarded-For $remote_addr;
+        proxy_redirect off;
+
+        proxy_pass http://app-server-upstream;
+    }
+
+    # Health check
+    location = /healthcheck/ {
+        proxy_set_header Host $http_host;
+        proxy_set_header X-Forwarded-For $remote_addr;
+        proxy_redirect off;
+
+        proxy_pass http://app-server-upstream;
+    }
+
+    # Static Assets
+    location / {
+        root /srv/dist;
+        index index.json index.html;
+
+        proxy_set_header Host $http_host;
```

Comment:
I'm not sure these need to be here, right? No `proxy_pass` being set in this block.